serverless.yml

```diff
@@ -0,0 +1,69 @@
+# Welcome to serverless. Read the docs
+# https://serverless.com/framework/docs/
+
+# Serverless.yml is the configuration the CLI
+# uses to deploy your code to your provider of choice
+
+# The `service` block is the name of the service
+service: hathor-explorer-service
+
+# The `provider` block defines where your service will be deployed
+provider:
+  name: aws
+  runtime: python3.8
+  profile: serverless
+  region: us-east-1
+  vpc:
+    securityGroupIds:
+      - sg-096e4b4e
```

Comment:
This id belongs to a security group in Hathor's AWS account?

Reply:
It's mine. I'll change to env var

Reply:
Changed it to env vars